api/user.go

```diff
@@ -1355,6 +1365,13 @@ func updateUser(c *Context, w http.ResponseWriter, r *http.Request) {
 			go sendEmailChangeUsername(c, rusers[1].Username, rusers[0].Username, rusers[0].Email, c.GetSiteURL())
 		}
 
+		updatedUser := rusers[0]
+		updatedUser.Sanitize(map[string]bool{})
```

Comment:
This should sanitize it like we do in https://github.com/mattermost/platform/blob/master/api/user.go#L1005 or else it'll potentially wipe out the user's email, first/last name, etc when they're updated. Same with the other place you sanitize the users. Perhaps we should just have a helper function that sanitizes based on the config and context since we seem to be duplicating this code in a few other places as well